Resources/Prototypes/Entities/Objects/Weapons/Melee/knife.yml

```diff
@@ -101,12 +101,16 @@
   id: SurvivalKnife
   description: Weapon of first and last resort for combatting space carp.
   components:
+  - type: Tag
+    tags:
+      - CombatKnife
+      - Knife
   - type: Sprite
     sprite: Objects/Weapons/Melee/survival_knife.rsi
     size: 2
     state: icon
   - type: Item
-    size: 10
+    size: 12
```

Comment:
wtf why you changed size

Reply:
I don't particularly remember changing the size, but this was over a week ago. It was likely to compensate for the buff the knife was given.